lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc

```diff
@@ -325,22 +325,22 @@ function dosomething_helpers_add_js_onboarding($nid) {
       'slides' => [],
     ];
 
-    if (variable_get('dosomething_campaign_onboarding_context_slide', FALSE)) {
-      $onboardingSettings['slides'][] = 'ContextSlide';
+    if (variable_get('dosomething_campaign_onboarding_reportback_items_slide', FALSE)) {
+      $onboardingSettings['slides'][] = 'ReportbackItemsSlide';
     }
 
     if (variable_get('dosomething_campaign_onboarding_instruction_slide', FALSE)) {
       $onboardingSettings['slides'][] = 'InstructionSlide';
     }
 
-    if (variable_get('dosomething_campaign_onboarding_reportback_items_slide', FALSE)) {
-      $onboardingSettings['slides'][] = 'ReportbackItemsSlide';
-    }
-
     if (variable_get('dosomething_campaign_onboarding_campaign_slide', FALSE)) {
       $onboardingSettings['slides'][] = 'CampaignSlide';
     }
 
+    if (variable_get('dosomething_campaign_onboarding_context_slide', FALSE)) {
+      $onboardingSettings['slides'][] = 'ContextSlide';
+    }
+
```

Comment:
But why not just reshuffle [here](https://github.com/DoSomething/phoenix/blob/96e1346d63e92d63068889d845b6664b6c76b860/lib/themes/dosomething/paraneue_dosomething/js/onboarding.js#L56)?